be/src/exec/workgroup/work_group.cpp

```diff
@@ -0,0 +1,60 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "exec/workgroup/work_group.h"
+
+#include "runtime/exec_env.h"
+
+namespace starrocks {
+namespace workgroup {
+WorkGroup::WorkGroup(const std::string& name, int id, size_t cpu_limit, size_t memory_limit, size_t concurrency,
+                     WorkGroupType type)
+        : _name(name),
+          _id(id),
+          _cpu_limit(cpu_limit),
+          _memory_limit(memory_limit),
+          _concurrency(concurrency),
+          _type(type) {
+    _mem_tracker = std::make_shared<starrocks::MemTracker>(memory_limit, name,
+                                                           ExecEnv::GetInstance()->query_pool_mem_tracker());
+    _driver_queue = std::make_unique<starrocks::pipeline::QuerySharedDriverQueue>();
```

Comment:
Should we init `_mem_tracker` and `_driver_queue` by Initializer List outside constructor body?

Reply:
ok